umake/frameworks/kotlin.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Canonical
+#
+# Authors:
+#  Omer Sheikh
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; version 3.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+
+"""Kotlin module"""
+
+from contextlib import suppress
+from gettext import gettext as _
+import logging
+import os
+import re
+import umake.frameworks.baseinstaller
+from umake.interactions import DisplayMessage
+from umake.tools import add_env_to_user
+from umake.ui import UI
+
+logger = logging.getLogger(__name__)
+
+
+class KotlinCategory(umake.frameworks.BaseCategory):
+
+    def __init__(self):
+        super().__init__(name="Kotlin", description=_("The Kotlin Programming Language"), logo_path=None)
+
+
+class KotlinLang(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Kotlin Lang", description=_("Kotlin language standalone compiler"),
+                         is_category_default=True, category=category, packages_requirements=["default-jre"],
```

Comment:
instead of "default-jre" put "openjdk-7-jdk | openjdk-8-jdk"
This way it will succeed if at least one of the two is installed